server/util/getRatingIDs.go

```diff
@@ -0,0 +1,16 @@
+package util
+
+import "github.com/factly/vidcheck/model"
+
+func GetRatingAndClaimantIDs(claims []model.ClaimData) ([]uint, []uint) {
```

Comment:
This function is not used anywhere in video/create and video/update you can use this.